docs/tutorials/fundamentals/installation.md

```diff
@@ -169,119 +166,142 @@ you're not using.
 
-![image: MacOS open bundle dialog
-box](../assets/tutorials/installation/bundle_open.png)
+You can now get started using napari! Use Launchpad to open the application. 
 
-After clicking "Open", the viewer should appear. Don't worry, you only have to
-go through this process once when you install a new bundle.
+![image: napari icon in MacOS laundpad](.../docs/images/bundle_13.png)
+
+```{note} 
+The first time you open napari you must use the Launchpad, but subsequently, the napari application will show up in Spotlight search.
+```
+
+napari comes installed with sample images from scikit-image. Use the dropdown menu File > Open Sample > napari to open a sample image, or open one of your own images using File > Open or dragging and dropping your image onto the canvas. 
+
+Next check out our [tutorial on the viewer](https://napari.org/tutorials/fundamentals/viewer.html) or explore any of the pages under the [Usage tab](https://napari.org/usage.html).
 
 ### Installing the Windows bundle
 
-Once you have downloaded the Windows bundle zip you will have a zip file with a
-name like `napari-0.3.7-Windows.zip`. Unzip the bundle (you may like to use a
-tool like [7-zip](https://www.7-zip.org/) for this) and double click on msi
-file, eg: `napari-0.3.7.msi`
+Once you have downloaded the Windows executable file, you will have a file with a name like `napari-0.4.15-Windows-x86_64.exe`. Double click this file to open the napari Setup Wizard. Click "Next" to continue.
+
+![image: napari Setup Wizard start page](.../docs/images/bundle_17.png)
+
+To continue, read and agree to the License Agreement by clicking ‘I Agree’.
+ 
+![image: napari License Agreement](.../docs/images/bundle_18.png)
 
-The napari setup wizard will then open. Click "Next" to begin the installation.
+The recommended installation method is to install napari just for the current user. However, you may install for all users using administrator privileges.
 
-![image: Windows napari setup
-wizard](../assets/tutorials/installation/windows_bundle_installer_start.png)
+![image: napari Setup Wizard user installation options](.../docs/images/bundle_19.png)
 
-![image: Windows napari setup wizard in
-progress](../assets/tutorials/installation/windows_bundle_installer_progress.png)
+Next you will be shown how much space will be used by the installation and the default destination folder, which can be updated using the ‘Browse’ button. Click ‘Next’ to continue.
 
-After the setup wizard has installed napari, click "Finish" to exit.
+![image: napari Setup Wizard installation location](.../docs/images/bundle_20.png)
 
-![image: Windows napari setup wizard
-finished](../assets/tutorials/installation/windows_bundle_installer_finish.png)
+On the next page, we recommend you check ‘Clear the package cache upon completion’ since this frees up memory in your machine’s cache without compromising napari functionality following installation. Click ‘Install’ to start the installation process.
```

Comment:
I'd avoid mentioning the "Clear the package cache" explicitly here. It's now available (and it's safe to use), but we might change that possibility in the future to deal with reinstallations, updates and other features.

```suggestion
On the next page, click ‘Install’ to start the installation process.
```